src/notifications/useActiveChannels.tsx

```diff
@@ -0,0 +1,29 @@
+import { NotificationCentre } from "./NotificationCentre"
```

Comment:
in questo file non c'è JSX, dovrebbe avere estensione `.ts`

Reply:
ah right